ding/model/template/vac.py

```diff
@@ -265,12 +265,17 @@ def compute_actor(self, x: torch.Tensor) -> Dict:
             >>> assert actor_outputs['logit'].shape == torch.Size([4, 64])
         """
         if self.share_encoder:
-            x = self.encoder(x)
+            # import ipdb;ipdb.set_trace()
```

Comment:
modify the corresponding API comments, and the `isinstance(x, dict)` to control the logic

Reply:
done